Common/Models/DeviceModelConstants.cs

```diff
@@ -1,55 +0,0 @@
-﻿namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Models
-{
-    public static class DeviceModelConstants
-    {
-        public const string OBJECT_NAME = "ObjectName";
```

Comment:
Also, can you check if these constants were being used for Json conversion ? In that case we may want to add JsonProperty to Common.Models.Device with these constants.